ci/management/commands/generate_reports.py

```diff
@@ -303,3 +352,68 @@ def handle_health_worker_registrations(
                 'Cadre': operator_details.get('role'),
                 'Number of Registrations': count,
             })
+
+    def handle_enrollments(self, sheet, sbm_client, ids_client, start_date,
+                           end_date):
+
+        sheet.set_header([
+            'Message set',
+            'Roleplayer',
+            'Currently enrolled (snapshot at point y)',
+            'Cumulatively enrolled (in last period x)',
+            'Of the cumulatively enrolled (in last period x), number which '
+            'opted out',
+            'Of the cumulatively enrolled (in last period x), the number which'
+            ' completed messages',
```

Comment:
For my implementation I've left out the things in brackets, since they seem to be explanations rather than something useful in the title, and I find them confusing. Up to you, but I would think that `Total enrolled`, `Enrolled in period`, `Enrolled and opted out in period`, and `Enrolled and completed in period` would be clearer?